.github/workflows/bindings.yml

```diff
@@ -332,16 +330,9 @@ jobs:
         uses: taiki-e/install-action@v2
         with:
           tool: wasm-pack
-      - uses: davidB/rust-cargo-make@v1
-      - name: build wasm
-        run: |
-          cd crypto-ffi
-          cargo make wasm
       - name: build ts
         run: |
-          cd crypto-ffi/bindings/js
-          bun install --frozen-lockfile
-          bun run build
+          make ts
```

Comment:
Similar comment as above, this is going to build wasm bindings in dev mode. We should generally set `WASM_RELEASE=1` in our CI jobs. Currently we're setting in only for publishing, but that means we're publishing something we didn't test in CI.